playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -439,3 +443,76 @@ def stack_overflow_enricher(alert: PrometheusKubernetesAlert):
             )
         ]
     )
+
+
+class ForeignLogParams(LogEnricherParams):
+    """
+    :var selectors: List of specific label selectors to retrieve logs from
+    """
+
+    selectors: List[str]
+
+
+@action
+def foreign_logs_enricher(event: ExecutionBaseEvent, params: ForeignLogParams):
+    """
+    Generic enricher to fetch and attach pod logs.
+
+    The logs are fetched for the pod determined by the label selector field in the parameters.
+    """
+
+    api = client.CoreV1Api()
+    matching_pods: List[V1Pod] = []
+    for selector in params.selectors:
+        try:
+            pods: V1PodList = api.list_pod_for_all_namespaces(label_selector=selector)
+            if pods.items:
+                matching_pods = pods.items
+                break
```

Comment:
this will only get the first selectors pods and not all of them. is that the behavior you want?
specifically with this issue there couldn't there be multiple prometheus operators so might want from all of them?
I would replace it with 
`matching_pods.extend(pods.items)
`

Reply:
So @arikalon1 and I discussed about this on the call. The idea here is to use the first matching label selector just like we do for finding prometheus urls.